apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/index.tsx

```diff
@@ -87,11 +82,18 @@ const SyncOnboardingScreen: React.FC<SyncOnboardingScreenProps> = ({
   const [toggleOnboardingEarlyCheckType, setToggleOnboardingEarlyCheckType] = useState<
     null | "enter" | "exit"
   >(null);
+  const [fwUpdateInterrupted, setFwUpdateInterrupted] = useState<FinalFirmware | null>(null);
```

Comment:
Pulling this state out of the early security check component because it gets unmounted when we restart the polling after the user restarts an update.